ra-optimisation/search-tree-rao/src/main/java/com/farao_community/farao/search_tree_rao/commons/RaoLogger.java

```diff
@@ -41,16 +45,19 @@ private RaoLogger() {
     }
 
     public static void logSensitivityAnalysisResults(String prefix,
-                                              ObjectiveFunction objectiveFunction,
-                                              PrePerimeterResult sensitivityAnalysisResult,
-                                              RaoParameters raoParameters,
-                                              int numberOfLoggedLimitingElements) {
+                                                     ObjectiveFunction objectiveFunction,
+                                                     Crac crac,
+                                                     Network network,
```

Comment:
It would be a cleaner signature to put  RangeActionActivationResult as an argument